frontend/js/redux/reducers/social-reducer.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+import {
+    UPDATE_SOCIAL_MEDIA_TAGS
+} from '../actions/social-actions';
+
+const baseUrl = 'http://okcandidate.code4hr.org';
+
+const initialState = {
+    siteName: 'OKCandidate',
+    description: 'Find out which candidates for local office are a match for you!',
+    image: `${baseUrl}/dist/images/ballot-box.png`,
+    shortDescription: 'Find out which candidates for local office are a match for you!',
+    creator: '@code4hr',
+    passPhrase: ''
+};
+
+function getDescription(matches) {
+    return `${getShortDescription(matches)} ${initialState.description}`;
+}
+
+function getShortDescription(matches) {
+    return `I matched with ${candidateMatchString(matches)}.`;
+}
+
+function candidateMatchString(matches) {
```

Comment:
you can simplify and make this function more generic by doing the following 😄 

return matches.slice(0,-1).join(', ') + `, and ${matches.slice(-1)[0]}`